### DISCUSSION TITLE
Add special #listing(s) tag for promotional posts

### DISCUSSION THREAD

**Discussion by cmgorton** [forem/forem#15467] [Icebox]
[Original post](https://forem.dev/codemouse92/add-special-listing-s-tag-for-promotional-posts-56d8) on forem.dev by Jason C. McDonald

Listings are, necessarily, quite short. However, many users of the DEV platform seem to feel they need more space to describe their product, service, or software, and this makes sense.

As a result, a number of users will publish promotional-focused DEV which, aside from violating the Terms of Use, also come across as "spam" to many readers.

In short, there are two problems:

1. Some users need to use the long format of a Post to add detail that won't fit in a Listing.
2. Said posts are annoying to many users when they appear in the normal feeds.

The #listings tag exists, at least on DEV.to, but was never given a formal purpose. I propose we repurpose it for promotional posts.

- Posts under #listings are forcibly removed from all feeds except its own. That is, a post may be tagged as listings python beginners, but the mere presence of listings will ensure it does not show up in the #python or #beginners feeds, nor in search.
- Users may include a link to their #listing post in their Listing: **this will be the only way that a #listings post can be viewed without directly having the link.

It would be important to alias #listing and #listings.

Some other things to consider, perhaps later:

- A message visible only to the post author, alerting them that listings are not shown in normal feeds, and prompting them to create a corresponding Listing.
- A canonical "read more" link field in Create Listings.

(Possibly) the ability of any tag moderator to add the listings tag to a post. That way, otherwise good promotional posts can be moved without requiring negative actions (Vomit, Downvote).

**The Alternative**
We could retool the Listings themselves to support longer format. However, this would replicate a lot of work already done on Posts, so it would probably be simpler to just connect the two.

**Comment by Steven13240** [comment]
Yeah your suggestion are highly appreciated. As we have did the same that the promotional product related to the [1pm results](https://alllotterysambad.com/) of dear lottory today. So that has increased the engagement of our business and are improving day by day. As the same I suggested also to one of my friend and when he did so he also detected improvement.
